test/support/flow_test_helper.rb

```diff
@@ -151,8 +151,26 @@ def outcome_body
       OutcomePresenter.new(flow.node(state.current_node_name), nil, state).body
     end
 
+    def outcome_title
+      raise "#{state.current_node_name} is not an outcome" unless current_node_type == :outcome
+
+      OutcomePresenter.new(flow.node(state.current_node_name), nil, state).title
+    end
+
     def outcome_body_text
       Nokogiri::HTML::DocumentFragment.parse(outcome_body).text
     end
+
+    def outcome_title_text
+      Nokogiri::HTML::DocumentFragment.parse(outcome_title).text
+    end
+
+    def outcome_text
+      "#{outcome_title}#{outcome_body}"
```

Comment:
```suggestion
      [outcome_title, outcome_body].reject(&:blank?).join("\n")
```

As this method is suffixed with text I think we should use the outcome_body_text in it so that it only returns text, the [outcome_title can only return text](https://github.com/alphagov/smart-answers/blob/06bda75726f104de6ceb25927415cbff16a7046b/lib/smart_answer/erb_renderer/format_capture_helper.rb#L3-L9).

I think a new line might be a reasonable separator for these so we can assert on it, but we need to only apply that if one of them isn't empty otherwise it'll blow the empty check.